### PR TITLE
improve tasks, support AndroidManifest override

### PR DESCRIPTION
**Pull request by AriaMoradi**
- simplifies running the generators
- `AndroidManifest.xml` override support for #5667

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt [hunk 1]
```diff
@@ -82,6 +86,7 @@ interface ThemeSourceGenerator {
             val overridesPath = "$userDir/multisrc/overrides" // userDir = tachiyomi-extensions project root path
             val resOverridesPath = "$overridesPath/res/$themePkg"
             val srcOverridesPath = "$overridesPath/src/$themePkg"
+            val manifestOverridesPath = "$overridesPath/manifest/$themePkg"
```
Comment (arkon) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt:89]: This is fine for now, but I feel like overrides are distributed across too many disparate folders. It'd probably make more sense to have something like:

```
overrides
|- [themeName]
  |- [sourceName]
    |- src
    |- res
    |- AndroidManifest.xml
  |- 
```

Comment (AriaMoradi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt:89]: that looks better indeed, but rearranging the files is something I'm not wiling to do!
If I could automate it with a script or sth, I'll pr it.

Comment (ObserverOfTime) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt:89]: Here's a quick bash script that will rearrange the files when ran from the overrides folder.
```bash
for d in {src,res}/*/*; do
    mkdir -p "${d#*/}"
    mv "$d" "$_/${d%%/*}"
done

rm -r src res
```

Comment (AriaMoradi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt:89]: You are a bit late, it's already done.

Comment (ObserverOfTime) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/ThemeSourceGenerator.kt:89]: ¯\\\_(ツ)\_/¯

**Pull request merged**